sdk/plugin_test.go

```diff
@@ -844,3 +845,22 @@ func TestListenSettings_Validate_Error(t *testing.T) {
 		assert.Equal(t, testCase.errCount, len(merr.Errors), merr.Error())
 	}
 }
+
+// TestRedactedJsonConfigOutput tests that we redact passwords when we dump
+// them to the logs. map[string]interface{} values will be replaced with
+// REDACTED if the lowercase map key containes "pass".
+func TestRedactedJsonConfigOutput(t *testing.T) {
```

Comment:
Can we this be moved to `utils_test.go`, since `RedactPasswords` is defined in `utils.go`?

Reply:
Moved